tests/unit-tests/test-class-lesson.php

```diff
@@ -478,4 +478,36 @@ public function testFirstPrerequisiteIsCorrect() {
 		$first_lesson = Sensei()->lesson::find_first_prerequisite_lesson( $course_with_lessons['lesson_ids'][3], $user_id );
 		$this->assertEquals( $course_with_lessons['lesson_ids'][1], $first_lesson );
 	}
+
+	public function testLessonsAssignedToACourseShouldHavePrerequisitesFromThatCourse() {
+		/* Arrange. */
+		$course_with_lessons   = $this->factory->get_course_with_lessons(
+			[
+				'lesson_count'   => 3,
+				'question_count' => 0,
+			]
+		);
+		$lesson_id_with_course = $course_with_lessons['lesson_ids'][0];
```

Comment:
I found this naming to be a bit confusing. Is this just the lesson ID?

Reply:
Yep, this made sense before I refactored the code. Changing it to `$lesson_id`.